README.md

```diff
@@ -138,6 +141,14 @@ are defined for local development:
 - User: `user`, password: `user` to browse books
 - User: `admin`, password: `admin` to manage books and orders
 
+### Testing in hybrid mode
+
+You can test the `GenreHierarchyTest` on H2 using the profile `default` as well as on HANA using the profile `hybrid`
```

Comment:
I'd rather recommend usage of existing profiles in application.yaml, e.g. `cloud`, which expects HANA.